iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -1063,6 +1067,7 @@ void PortManager::removePublisherFromServiceRegistry(const capro::ServiceDescrip
 
 void PortManager::addServerToServiceRegistry(const capro::ServiceDescription& service) noexcept
 {
+    std::cout << "#### " << service << std::endl;
```

Comment:
Is this some kind of printf debugging artifact ;)

Reply:
upsi